src/components/code_view.py

```diff
@@ -358,7 +361,33 @@ def __init__(self, master, parent, **kwargs):
         for file in self.file_names:
             self.create_new_tab(file)
             self.bind_esc(editor=self.code_editors[file], file_name=file)
-    
+
+    def quick_run(self):
+        active_editor = self.editor
+        if active_editor:
+            # Ensure the latest text is always fetched and transpiled
+            source_code = active_editor.text.get('1.0', 'end-1c')  # Fetching the latest code from the editor
+            active_editor.update_source_code(source_code)  # Method to update the editor's source code attribute if needed
+
+            # Run lexer, parser, and analyzer every time to reflect latest changes
+            lx_res = active_editor.run_lexer()
+            p_res = active_editor.run_parser()
+            a_res = active_editor.run_analyzer()
+
+            # Update analyzer tabs and compiler logs
+            self.parent.master.update_analyzer_tabs_and_logs(active_editor, lx_res, p_res, a_res)
+
+            if lx_res and p_res and a_res:
+                if active_editor.transpiled_program:
+                    compiler = Compiler(py_source=active_editor.transpiled_program, filename=active_editor.filename)
+                    compiler.run_python()  # Run the Python transpiled code
```

Comment:
this is correct, `quick_run` should be getting the transpiled program, verifying, and running it sa cmd.

evaluation should be done before this is called. try to limit na ang nangyayari lang sa quick run is this